lts.json

```diff
@@ -21,21 +21,21 @@
   "ember-cli-sri": "^2.1.0",
   "ember-cli-test-loader": "^1.1.0",
   "ember-cli-uglify": "^1.2.0",
-  "ember-computed-decorators": "~0.2.0",
-  "ember-concurrency": "~0.7.0",
+  "ember-computed-decorators": "~0.2.2",
+  "ember-concurrency": "~0.7.15",
   "ember-css-modules": "~0.5.0",
   "ember-data": "~2.8.0",
   "ember-data-factory-guy": "^2.0.0",
   "ember-data-model-fragments": "~2.3.0",
   "ember-export-application-global": "^1.0.5",
   "ember-get-config": "~0.1.0",
-  "ember-hook": "^1.0.0",
+  "ember-hook": "^1.3.5",
   "ember-ignore-children-helper": "^1.0.0",
   "ember-intl": "^2.0.0",
   "ember-load-initializers": "^0.5.1",
   "ember-local-resolver": "^1.0.0",
-  "ember-lodash-shim": "^1.0.0",
-  "ember-mocha": "~0.10.0",
+  "ember-lodash-shim": "^1.0.1",
+  "ember-cli-mocha": "~0.11.0",
   "ember-moment": "^6.0.0",
   "ember-page-title": "~0.3.0",
   "ember-phoenix": "~0.1.0",
```

Comment:
Is this being used in our internal websocket addon?  I'm not quite sure why we would list something like this in the LTS set.  Basically anything that is going to be exclusively referenced by a top level addon in the LTS doesn't seem like it belongs here.